sys/log/full/src/log_shell.c

```diff
@@ -251,6 +276,29 @@ shell_log_dump_cmd(int argc, char **argv)
             continue;
         }
 
+#if MYNEWT_VAL(LOG_FCB_BOOKMARKS)
+        if (dump_bmarks) {
+            bmarks = log_fcb_get_bmarks(log, &bmarks_size);
+            for (i = 0; i < bmarks_size; i++) {
+#if MYNEWT_VAL(LOG_FCB)
+                console_printf("%u: index:%lu entry: %x fa: %x fe_elem_off: %lx\n", i,
+                               bmarks[i].lfb_index,
+                               (uintptr_t)&bmarks[i].lfb_entry,
+                               (uintptr_t)bmarks[i].lfb_entry.fe_area,
```

Comment:
more useful would be to print offset in flash instead of address of array element

Reply:
Updated version will not compile due `.` instead of `->`